src/order/order_dual.lean

```diff
@@ -69,4 +69,9 @@ lemma to_dual_le [has_le α] {a : α} {b : order_dual α} :
 lemma to_dual_lt [has_lt α] {a : α} {b : order_dual α} :
   to_dual a < b ↔ of_dual b < a := iff.rfl
 
+/-- Recursor for `order_dual α`. -/
+@[elab_as_eliminator]
+def dual_rec {C : order_dual α → Sort*} (h₂ : Π (a : α), C (to_dual a)) :
```

Comment:
Possibly
```suggestion
protected def rec {C : order_dual α → Sort*} (h₂ : Π (a : α), C (to_dual a)) :
```
Since `order_dual.dual_rec` is a bit repetitive